File: Python/Assignments/counter.py
"""

"""

# print("Please enter the number of coins")

inp_quarters = input()
inp_dimes =  input()
inp_nickles = input()
inp_pennies = input()

totalDollars = float(inp_quarters)/4 + float(inp_dimes)/10 + float(inp_nickles)/20 + float(inp_pennies)/100
totalcent = float("{:.2f}".format(totalDollars))  -  float("{:.0f}".format(totalDollars)) 

print("The total is", "{:.0f}".format(totalDollars), "dollars and", "{:.2f}".format(totalcent).replace('0.','') ,"cents")

File: C++/Assignments/counter.cpp
#include <iostream>
using namespace std;


int main()
{

double inp_quarters, inp_dimes, inp_nickles, inp_pennies;
double total;

cout <<  "Please enter the number of coins: "<<endl;
cout << "# of quarters: ";
cin >> inp_quarters;
cout << "# of dimes: ";
cin >> inp_dimes;
cout << "# of nickels: ";
cin >> inp_nickles;
cout << "# of pennies: ";
cin >> inp_pennies; 

total = inp_quarters*.25 + inp_dimes*.10 + inp_nickles*.05 + inp_pennies *.01;

cout <<"The total is " << int(total) << " dollars and "<< (total-int(total))*100<<" cents"<<endl;

return 0;
}
